pages/draft-06/json-hyper-schema-release-notes.md

```diff
@@ -1,107 +1,104 @@
----
-title: JSON Hyper-Schema Draft-06 Release Notes
-section: docs
----
+### JSON Hyper-Schema Draft-06 Release Notes
```

Comment:
Why did we remove the front matter in these files?

Reply:
Using the type format, we agreed to "type 4" (a coalition of types 2 and 3), which means everything in the draft six folder would be rendered from the index.page.tsx file. It is why I didn't use the front matter in the individual .md file.